utilities/wal_filter.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#include "rocksdb/wal_filter.h"
+
+#include <memory>
+
+#include "rocksdb/convenience.h"
+#include "rocksdb/options.h"
+#include "rocksdb/utilities/customizable_util.h"
+
+namespace ROCKSDB_NAMESPACE {
+Status WalFilter::CreateFromString(const ConfigOptions& config_options,
```

Comment:
If it is just the CreateFromString, can we move it to wal_filter.h? Declaration+definition.

Reply:
> If it is just the CreateFromString, can we move it to wal_filter.h? Declaration+definition.

It should really be here.  If there is a default or "shipped" implementation later, it should be registered here.